core/metadata.js

```diff
@@ -109,14 +133,15 @@ export type Metadata = {
   overwrite?: Metadata,
   childMove?: boolean,
   path: string,
+  local: MetadataLocalInfo,
```

Comment:
As I understand, `local` cannot be null or undefined (no `?`). So, why are you comparing it to `null` on https://github.com/cozy-labs/cozy-desktop/pull/1904/files#diff-6d18f9e7a792437c06b6f95186c97b88R322 ?

Reply:
It can actually be null, if the document is built by the remote watcher during a remote file addition.
However, marking it as potentially null or undefined means tons of unnecessary checks with Flow… I chose to go with the same pattern as for `remote` which can in fact be null while the type says it can't.